lib/Graph/Graph.cpp

```diff
@@ -965,8 +965,15 @@ ARITHMETIC_FUN_DEF(Max);
 ARITHMETIC_FUN_DEF(Min);
 #undef ARITHMETIC_FUN_DEF
 
-/// \returns the result type for a logical operation.
-static TypeRef getResultTypeOfLogicalOp(Module &M, TypeRef T) {
+/// This helper function is used only by the functions creating boolean
+/// operations like createCmpEQ and createCmpLTE to compute their output types.
+/// The output type is computed based on the type \p T of the operand of the
+/// logical operation. In case of a quantized type we provide concrete scale and
+/// offset for the type as we know that the output of a boolean operation could
+/// be only 0 or 1.
+///
+/// \returns the output type for a boolean operation.
+static TypeRef getResultTypeOfBooleanOp(Module &M, TypeRef T) {
```

Comment:
consider renaming in the GraphOptimizer as well (LOGICAL_OP_CASE) to be consistent with this naming schema.

Reply:
Done.